x/src/test.rs

```diff
@@ -100,6 +153,38 @@ pub fn run(mut args: Args, config: Config) -> Result<()> {
             all_features,
             ..base_args
         })?;
-        Ok(())
     }
+
+    let mut debug_dir = project_root();
+    debug_dir.push("target/debug/");
+
```

Comment:
This can be moved into the if statement below

Reply:
done.